### ISSUE TITLE
Fails to build using lld

### ISSUE THREAD

**Issue by mssun** [japaric/steed#156]
I followed the using `lld` section in the README but failed with some error.

```
➜  hello (master) ✗  sudo apt-get install lld-4.0
➜  hello (master) ✗ curl -LO https://raw.githubusercontent.com/japaric/steed/master/docker/x86_64-unknown-linux-steed.json
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   637  100   637    0     0   2397      0 --:--:-- --:--:-- --:--:--  2403
➜  hello (master) ✗ ls
Cargo.lock  Cargo.toml  src  x86_64-unknown-linux-steed.json
➜  hello (master) ✗ xargo run --target x86_64-unknown-linux-steed
error: failed to run `rustc` to learn about target-specific information

Caused by:
  process didn't exit successfully: `rustc - --crate-name ___ --print=file-names --sysroot /home/mssun/.xargo -Z force-unstable-if-unmarked --target x86_64-unknown-linux-steed --crate-type bin --crate-type rlib` (exit code: 101)
--- stderr
error: Error loading target specification: Field target-c-int-width in target specification is required
  |
  = help: Use `--print target-list` for a list of built-in targets


error: `"cargo" "build" "--release" "--manifest-path" "/tmp/xargo.MCOR8rrKDKUA/Cargo.toml" "--target" "x86_64-unknown-linux-steed" "-p" "core"` failed with exit code: Some(101)
note: run with `RUST_BACKTRACE=1` for a backtrace
➜  hello (master) ✗ xargo --version
xargo 0.3.8
cargo 0.23.0-nightly (e447ac7e9 2017-09-27)
```

**Comment by japaric** [comment]
Thanks for the heads up. This is due to a breaking change in the target specification from a recent nightly change. #157 should fix this.

**Comment by japaric** [comment]
This particular error was fixed in #157 so I'm going to close this issue. Let me know if you run into any other problem!
